Source/WebKit/UIProcess/API/C/WKPage.cpp

```diff
@@ -3287,3 +3287,14 @@ void WKPageSetTopContentInsetForTesting(WKPageRef pageRef, float contentInset, v
         callback(context);
     });
 }
+
+void WKPageSetPageScaleFactorForTesting(WKPageRef pageRef, float scaleFactor, WKPoint point, void* context, WKPageSetPageScaleFactorForTestingFunction completionHandler)
+{
+    auto* pageForTesting = toImpl(pageRef)->pageForTesting();
+    if (!pageForTesting)
```

Comment:
I'm not sure if we want this case since it's for testing and this should never happen.  Fine either way.